flex-templates/java/regional_dlp_transform/metadata.json

```diff
@@ -0,0 +1,58 @@
+{
+  "name": "Streaming BiqQuery data through DLP to reidentify data flex template",
+  "description": "Java flex template for streaming BiqQuery data through regional DLP to reidentify the data.",
+  "parameters": [
+    {
+      "name": "inputBigQueryTable",
+      "label": "BigQuery input table name.",
+      "helpText": "Input BigQuery table for results specified as: PROJECT:DATASET.TABLE ."
+    },
+    {
+      "name": "outputBigQueryTable",
+      "label": "BigQuery output table name.",
+      "helpText": "Output BigQuery table with transformed data as: PROJECT:DATASET.TABLE ."
+    },
+    {
+      "name": "deidentifyTemplateName",
+      "label": "DLP structured de-identification template name.",
+      "helpText": "Name of the DLP structured de-identification template that was used to deidentify data in the BiqQuery source. Format is 'projects/<PROJECT>/locations/<LOCATION>/deidentifyTemplates/<TEMPLATE_ID>'",
+      "regexes": [
+        "projects/[^/]+/locations/[^/]+/deidentifyTemplates/[-_a-zA-Z0-9]+"
+      ]
+    },
+    {
+      "name": "dlpLocation",
+      "label": "DLP location.",
+      "helpText": "The location of the DLP template resource."
+    },
+    {
+      "name": "dlpProjectId",
+      "label": "DLP project.",
+      "helpText": "ID of the project that holds the DLP template."
+    },
+    {
+      "name": "confidentialDataProjectId",
+      "label": "Project for output BigQuery that has confidential data.",
+      "helpText": "ID of the project that holds the confidential data in BigQuery."
+    },
+    {
+      "name": "dlpTransform",
+      "label": "DLP transformation type.",
+      "helpText": "Choose between DLP DE-IDENTIFY or RE-IDENTIFY.",
+      "regexes": [
+        "RE-IDENTIFY|DE-IDENTIFY"
+      ]
+    },
+    {
+      "name": "batchSize",
+      "label": "DLP batch size.",
+      "helpText": "Number of records to be sent in a batch in the call to the DLP API. Default is 100",
+      "isOptional": true
+    },
+    {
+      "name": "bqSchema",
+      "label": "BigQuery table schema.",
+      "helpText": "Output BigQuery table schema specified as string with format: FIELD_1:STRING,FIELD_2:STRING,..."
+    }
```

Comment:
At each triggeringFrequency duration, a BigQuery load job will be generated for all data saved since the last load job. However, it is only necessary when reading *unbounded* data, i.e. data that is not in a structure like a table, and using the write method *FILE_LOADS*, which is the *de-id* case. Even the input is a csv, and has a pattern, it is a bucket text file and not a structure like a BQDataset. 
 In the *de-id* code, if we didn't use triggeringFrequency, we get the error:
```
"java.lang.IllegalArgumentException: When writing an unbounded PCollection via FILE_LOADS or STORAGE_API_WRITES, triggering frequency must be specified"
```
 For *re-id* code, which reads *bounded* data (BQDataset), this attribute is not necessary.